### PR TITLE
Form branch

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

File: components/form.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.form-label {
+  color: var(--drak-theme-color);
+}
+
+.form-field {
+  border: 1px solid var(--light-color);
+  width: 100%;
+  margin: 0.5rem 0;
```
Comment (iRohitGaur) [inline components/form.css]: give px for values less than 10px

Comment (zoef-yt) [inline components/form.css]: use px

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

**Review by sudipkundu999 — APPROVED** [review]
Code Looks good!
A suggestion related to 3rd input : 
prabhat
Make the form fields start at the point it'll look consistent

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Pull request merged**